dotnet/src/Connectors/Connectors.OpenAI/AzureSdk/ClientCore.cs

```diff
@@ -571,8 +573,10 @@ await foreach (StreamingChatCompletionsUpdate update in response.ConfigureAwait(
 
             // Add the original assistant message to the chatOptions; this is required for the service
             // to understand the tool call responses.
-            chatOptions.Messages.Add(GetRequestMessage(streamedRole ?? default, content, toolCalls));
-            chat.Add(new OpenAIChatMessageContent(streamedRole ?? default, content, this.DeploymentOrModelName, toolCalls, metadata));
+            chatOptions.Messages.Add(GetRequestMessage(streamedRole ?? default, content, streamedName, toolCalls));
+#pragma warning disable SKEXP0001 // Assigning nullable property self-consistent for internal/private code.
```

Comment:
I tried to remove this suppression and code builds without warnings. Do we still need it? Same in other places.

Reply:
We will...I think I dropped a property assignment.  Good eye.

Reply:
Ok, yeah...its in the project file is all.

Reply:
Cleaned